app/services/work_preservation_service.rb

```diff
@@ -73,15 +74,19 @@ def preservation_directory
       Pathname.new(@path).join("#{PRESERVATION_DIRECTORY}/")
     end
 
-    def s3_client
-      @work.s3_query_service.client
+    def s3_query_service
+      @s3_query_service ||= S3QueryService.new(@work, false, true)
```

Comment:
Then this would be a mode
```suggestion
      @s3_query_service ||= S3QueryService.new(@work, "preservation")
```